packages/plugin-ext/src/common/plugin-protocol.ts

```diff
@@ -390,7 +390,7 @@ export interface PluginModel {
  * This interface describes some static plugin contributions.
  */
 export interface PluginContribution {
-    configuration?: PreferenceSchema | PreferenceSchema[];
```

Comment:
Should not it be reverted? in https://github.com/Cmacu/gotoanything/blob/e17392ace574442e8fc107cd6ecfa7e0eb840f20/package.json#L70 it is an object, not an array

Reply:
It is handled before this level. I decided to not leak these array | object types deeper than necessary.